### PR TITLE
Update the portfolio page based on week2 walkthrough doc

### PR DESCRIPTION
**Pull request by lzylucy**
Update the portfolio page according to week2 walkthrough doc.
Still need to figure out the project ID and deploy the project on a live server.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -8,11 +8,58 @@
...
+
+      <h3>Experiences</h3>
+      <ul>	
+        <li>Research Assistant @ Columbia University (Sep 2019 - May 2020): <br/>
+          <div class = "description">Implement MalGAN to attack PDF malware classifiers. Github Link here: <a href = "https://github.com/xiaoluLucy814/Malware-GAN-attack">Malware-GAN-attack</a></div>
+          <br style = "line-height:5px;">
+
+        <li>Software Engineer @ Lamont-Doherty Earth Laboratory (Nov 2019 - May 2020) <br/>
+          <div class = "description">Working on <b>NYC Air Pollution Project</b></div>
+          <br style = "line-height:5px;">
```
Comment (zixiyan) [inline portfolio/src/main/webapp/index.html]: Per https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, please move styling to style sheet.

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -8,11 +8,58 @@
...
+          to join Google this summer!
+
+      <p>My interest in CS lies in ML and its applications.
+         I worked as a research assistant in my sophomore year when I implemented ML based attacks
+         to evade PDF malware classifiers.
+
+      <h3>Experiences</h3>
+      <ul>	
+        <li>Research Assistant @ Columbia University (Sep 2019 - May 2020): <br/>
+          <div class = "description">Implement MalGAN to attack PDF malware classifiers. Github Link here: <a href = "https://github.com/xiaoluLucy814/Malware-GAN-attack">Malware-GAN-attack</a></div>
```
Comment (zixiyan) [inline portfolio/src/main/webapp/index.html]: Curious, is it possible to split it into multiple lines (each line less than 80 characters)?

**Comment by floatfeather** [comment]
Could you please attach a screenshot of your UI?

**Comment by lzylucy** [comment]
Sure! 
UI for the home page (index.html)
homepage


UI for the photo gallery (photo.html)
photo

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (lzylucy) [reply portfolio/src/main/webapp/index.html]: yes! thanks for pointing out

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (lzylucy) [reply portfolio/src/main/webapp/index.html]: sure!

**Review by floatfeather — APPROVED** [review]

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -8,11 +8,78 @@
...
+        <li><b>THE Hack 2019:</b><br>
+        <div class = "description">
+            A project that scrapes stories for kids from online sources
+            using Scrapy and recommends stories based on preferences and 
+            feedback set by parents.</div>
+      </ul>
+
+      <a href="photo.html">
+        <img src="images/sunflower.png" alt="sunflower" 
+        style="width:350px;height:400px;">
```
Comment (zixiyan) [inline portfolio/src/main/webapp/index.html]: per https://google.github.io/styleguide/htmlcssguide.html#Separation_of_Concerns, move this to style sheet?

File: portfolio/src/main/webapp/photo.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <title>Photo Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Welcome to Sunflower's Photo Gallery</h1>
+      <p>Here's a collection of my favorite photos. Enjoy!
+      <div class="image-thumbnails">
```
Comment (zixiyan) [inline portfolio/src/main/webapp/photo.html:12]: Please format these elements accordingly. https://google.github.io/styleguide/htmlcssguide.html#HTML_Formatting_Rules.

Comment (xiaoluLucy814) [reply portfolio/src/main/webapp/photo.html:12]: I'm a bit not sure how to change this. I did not intend to make <div class="image-thumbnails"> a child of <p>, will adding an end tag </p> on line 11 help clarify the structure?

Comment (lzylucy) [reply portfolio/src/main/webapp/photo.html:12]: I'm a bit not sure how to change this. I did not intend to make ``<div class="image-thumbnails">`` a child of ``<p>``, will adding an end tag ``</p>`` on line 11 help clarify the structure?

**Comment by zixiyan** [comment]
I can't find your comment, could you please point me to the comment link?

Basically, the format rules suggest that you should indent the child
elements or just make them inline (less than 80 characters).

And actually I mean
<a ...>  </a> or
<a ...>
  
</a>

**Comment by lzylucy** [comment]
Here's the link:
https://github.com/xiaoluLucy814/STEP/pull/2#discussion_r439192664
I got what you mean and just reformat the code:)

File: portfolio/src/main/webapp/photo.html (hunk 4 above)
Comment (zixiyan) [reply portfolio/src/main/webapp/photo.html:12]: Actually I mean
```
<a ...> <img ...> </a>
```
or
```
<a ...>
  <img ...>
</a>
```

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -1,9 +1,43 @@
...
+#funfact-container {
+  margin-top:10px;
+  margin-bottom: 20px;
+}
+
+br{
+  line-height: 2px;
+}
+
+.image-link img{
```
Comment (zixiyan) [inline portfolio/src/main/webapp/style.css]: new space?
```
.image-link img {
```

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -8,11 +8,78 @@
...
+
+      <h3>Experiences</h3>
+      <ul>	
+        <li>Research Assistant 
+            @ Columbia University (Sep 2019 - May 2020):<br>
+          <div class = "description">
+              Implement MalGAN to attack PDF malware classifiers. 
+              Github Link here: 
+              <a href = "https://github.com/xiaoluLucy814/Malware-GAN-attack">
+              Malware-GAN-attack</a></div>
```
Comment (zixiyan) [inline portfolio/src/main/webapp/index.html]: Consider to format this block?

**Review by zixiyan — CHANGES_REQUESTED** [review]

File: portfolio/src/main/webapp/index.html (hunk 6 above)
Comment (lzylucy) [reply portfolio/src/main/webapp/index.html]: Is it okay to exceed 80 characters per line if the link is long? I'm trying to put ``<a>`` and ``</a>`` on the same line

Comment (zixiyan) [reply portfolio/src/main/webapp/index.html]: Yes. It's okay to exceed 80 characters if you have no other choice.
I actually means you should format the block like:
```
<div>
   ...
</div>
```

**Review by zixiyan — APPROVED** [review]

**Pull request merged**